production/helm/loki/templates/networkpolicy.yaml

```diff
@@ -3,24 +3,17 @@ apiVersion: networking.k8s.io/v1
 kind: NetworkPolicy
 metadata:
   name: {{ template "loki.fullname" . }}
-  namespace: {{ .Release.Namespace }}
```

Comment:
I believe people rely on this namespace being there so they can use `helm template` locally and install the output of that use kubectl. Should be added back. Even better would be to have a helper function for it so users can override it.